postgres/go.mod

```diff
@@ -3,8 +3,15 @@ module github.com/stealthrocket/net/postgres
 go 1.20
 
 require (
-	github.com/lib/pq v1.10.9
+	github.com/jackc/pgx/v5 v5.4.1
 	github.com/stealthrocket/net v0.1.3-0.20230609185417-24d7c1a5dd46
 )
 
+require (
+	github.com/jackc/pgpassfile v1.0.0 // indirect
+	github.com/jackc/pgservicefile v0.0.0-20221227161230-091c0ba34f0a // indirect
+	golang.org/x/crypto v0.9.0 // indirect
+	golang.org/x/text v0.9.0 // indirect
+)
+
 replace github.com/lib/pq => github.com/stealthrocket/pq v0.0.0-20230609191514-73472796856a
```

Comment:
I think we can remove this replace now too?

Reply:
Thanks! I missed it :)